Automattic/jetpack
# VideoPress: add title aria to player iframe

Fixes # 1097-gh-Automattic/greenhouse

#### Changes proposed in this Pull Request:

* title and aria-label attributes added to rendering of iframe for videopress/wpvideo shortcode uses
* oembed handled by wpcom D74271-code

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


nope

#### Testing instructions:






* apply pr
* add a `wpvideo` and `videopress` shortcode to a post
* publish the post then view the page
* inspect the source, both video iframes should now have `title` and `aria-label` attributes set on them
* If testing the `/video` block or oembed, D74271-code needs to be applied to wpcom sandbox with public-api sandboxed and Jetpack site needs to point to your sandbox as well (Jetpack pointing at sandbox may not be necessary)
